### PR TITLE
Update default branch to main

### PR DESCRIPTION
**Pull request by soapy1**

### CODE REVIEW

File: scripts/website_push_docs.sh [hunk 1]
```diff
@@ -34,7 +34,7 @@ git add .
 git commit -q -m "Deploy by $USER"
 
 git remote add heroku git@heroku.com:vagrantup-docs-2.git
-git push -f heroku master
+git push -f heroku main
```
Comment (soapy1) [inline scripts/website_push_docs.sh:37]: I think this should work (https://devcenter.heroku.com/changelog-items/1829) but should give it a go anyway

Comment (chrisroberts) [reply scripts/website_push_docs.sh:37]: This script is deprecated anyway and we can just remove :smiley:

**Review by chrisroberts — APPROVED** [review]
:tada:

File: scripts/website_push_docs.sh (hunk 1 above)
Comment (soapy1) [reply scripts/website_push_docs.sh:37]: oooo! how about `scripts/website_push_www.sh`?

**Pull request merged**